build.sbt

```diff
@@ -60,6 +61,7 @@ lazy val macroLib = Project("daffodil-macro-lib", file("daffodil-macro-lib"))
   .disablePlugins(OsgiCheckPlugin)
 
 lazy val propgen = Project("daffodil-propgen", file("daffodil-propgen"))
+  .disablePlugins(SbtXjcPlugin)
```

Comment:
Can this disable just become part of commonSettings, then the only clutter would be the one enableSettings for tdml?

Reply:
We did something similar - set the Seq containing xsd schemas to run through JAXB to empty in commonSettings - it allows us to remove all of these disablePlugins lines. It builds for me locally, so, hopefully, the CI tests will build now as well.